sdk/core/azure-core/azure/core/pipeline/base.py

```diff
@@ -37,6 +54,49 @@
 PoliciesType = List[Union[HTTPPolicy, SansIOHTTPPolicy]]
 
 
+def use_distributed_traces(func):
```

Comment:
We can probably simplify the name of this method. And if this is a general-purpose decorator (which I think it is, right?), it should live somewhere other than in `azure.core.pipeline`. `azure.core.tracing`?

Reply:
Should I also move `azure-sdk-for-python\sdk\core\azure-core\azure\core\pipeline\distributed_tracing\` outside of the pipeline as well?

Reply:
Yes. It is more than a pipeline concept.

Reply:
done: d040095a6053c200e61e92245a52c25fe91b9411